src/views/product-tutorials-view/components/product-view-content/components/tutorials-stack/types.ts

```diff
@@ -0,0 +1,74 @@
+import { Collection, ProductOption } from 'lib/learn-client/types'
+
+/**
+ * TODO
+ * Many of these types were dropped in from the existing front-end repo.
+ * Need to update to use the analogous types from lib/learn-client/types.
+ * Intent is to make those updates during implementation of TutorialsStack
+ */
+
+interface BaseTutorial {
```

Comment:
Perhaps we can lean on the client type for this similar to how you are `pick`ing the Collection.

Reply:
✅ I dig that, updated to take this approach!